packages/playground/src/components/pull-request/utils.tsx

```diff
@@ -0,0 +1,47 @@
+export type EnumPullReqReviewDecision = 'approved' | 'changereq' | 'pending' | 'reviewed'
```

Comment:
this file should be a `.ts` file.. not a `.tsx` file. we dont want to allow rendering react in util functions

Reply:
done